Apps/WebClient/src/ClientApp/src/store/modules/vaccinationStatus/actions.ts

```diff
@@ -12,6 +12,11 @@ import ErrorTranslator from "@/utility/errorTranslator";
 
 import { VaccinationStatusActions } from "./types";
 
+const retryStatusMessage =
+    "We're busy but will continue to try to fetch your record....";
+
+const retryLoggerMessage = "Re-querying for vaccination status";
```

Comment:
This message can be used when fetching the vaccination status, but we should use a different message when fetching the proof of vaccination.

Reply:
@MikeLyttle  I just changed the message on each actions